src/services/CLIService.ts

```diff
@@ -88,7 +88,7 @@ export class CLIService implements IService{
         const accounts = argv.accounts as number;
         const async = argv.async as boolean;
         const balance = argv.balance as number;
-        const detached = argv.detached as boolean;
+        const detached = argv.detached as boolean || true;
```

Comment:
I am a bit confused .. Why do we need that ?

Reply:
It's undefined otherwise and starts the tests in attached mode, which is not what we want.

Reply:
Then we can deffine a default value to the flag itself...if I understand correctly.

Just to double-check. Won't this override the flags behaviour. Even if I pass false it will be set to true

Reply:
Yes, you're right. I set it just as a way to workaround the issues in the test for the time being, didn't mean for this to be final. Should be changed